catalog/spatial/wfs/spatial-wfs-common/src/main/java/org/codice/ddf/spatial/ogc/wfs/catalog/common/FeatureMetacardType.java

```diff
@@ -390,4 +391,40 @@ public boolean isQueryable(String propertyName) {
   public List<String> getTemporalProperties() {
     return temporalProperties;
   }
+
+  public Set<String> getNonQueryableProperties() {
+    return Collections.unmodifiableSet(nonQueryableProperties);
+  }
+
+  public String getGmlNamespace() {
+    return gmlNamespace;
+  }
+
+  public XmlSchema getSchema() {
+    return schema;
+  }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (!(o instanceof FeatureMetacardType)) {
+      return false;
+    }
+    if (!super.equals(o)) {
+      return false;
+    }
+    FeatureMetacardType that = (FeatureMetacardType) o;
+    return featureType.equals(that.getFeatureType())
+        && nonQueryableProperties.equals(that.getNonQueryableProperties())
+        && gmlNamespace.equals(that.getGmlNamespace())
+        && schema.equals(that.getSchema());
```

Comment:
❓ Are these the right fields to include?

Reply:
Personally I would have included all of the fields:
```
properties
textualProperties
gmlProperties
temporalProperties
featureType
nonQueryableProperties
gmlNamespace;
```

Reply:
Looks like we populate the `properties`, `textualProperties`, `gmlProperties`, and `temporalProperties` from the schema, so I think it makes to just check the schema so long as equality guarantees that the above also are equal. Do you know if that's guaranteed?

Reply:
I don't know if it's guaranteed 🤷‍♀️ I'm going to stick with this for now because it's low risk.